pkg/cache/clusterqueue.go

```diff
@@ -333,11 +346,22 @@ func (c *clusterQueue) updateWithAdmissionChecks(checks map[string]AdmissionChec
 				inactive = append(inactive, acName)
 			}
 			checksPerController[ac.Controller] = append(checksPerController[ac.Controller], acName)
-			if ac.SingleInstanceInClusterQueue {
+
+			if features.Enabled(features.AdmissionCheckValidationRules) {
```

Comment:
Similarly here, avoid the diff by using dedicated cache fields for the new checks